src/mlpack/methods/dbscan/dbscan_impl.hpp

```diff
@@ -78,15 +109,20 @@ size_t DBSCAN<RangeSearchType, PointSelectionPolicy>::Cluster(
       ++counts[assignments[i]];
     }
   }
+  #endif
 
-  // We should be guaranteed that the number of clusters is always greater than
-  // zero.
+  // Normalize centroids
+  #ifdef MLPACK_USE_OPENMP
+  #pragma omp parallel for
+  #endif
   for (size_t i = 0; i < numClusters; ++i)
-    centroids.col(i) /= counts[i];
+    if (counts[i] > 0)
```

Comment:
Why remove the comment pointing out that we are guaranteed the number of points in a cluster is greater than 0?  (Then this check is not needed.)